### ISSUE TITLE
How to  ensemble pointclip and other point model?

### ISSUE THREAD

**Issue by IzhiSu** [ZrrSkywalker/PointCLIP#14]
Hello author.I am very interested in your research.Could you tell me how to ensemble pointclip and other point model such as pointnet?
Could you give me some tips?I'm looking forward to you reply.

**Comment by ZrrSkywalker** [comment]
Thanks for your interest! 
We directly transfer the code of PointCLIP into the code base of different methods, e.g. SimpleView's and CurveNet's. More simply, you can cache the predicted logits of PointCLIP and add them with other methods' predictions.
